docusaurus/docs/quickstarts/network/hosted.md

```diff
@@ -140,29 +138,78 @@ ubuntu@ip-10-0-0-1:~$ sudo systemctl -q status ziti-router --lines=0 --no-pager
              └─2385 /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/ziti-bin/ziti-v0.22.11/ziti-router run /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/ip-10…
 ```
 
+Inspect the servers' listening ports.
+
+```bash
+$ sudo ss -lntp | grep ziti
+LISTEN 0      4096               *:8440             *:*    users:(("ziti-controller",pid=5240,fd=10))
+LISTEN 0      4096               *:8441             *:*    users:(("ziti-controller",pid=5240,fd=9)) 
+LISTEN 0      4096               *:8442             *:*    users:(("ziti-router",pid=5272,fd=9))     
+LISTEN 0      4096               *:10080            *:*    users:(("ziti-router",pid=5272,fd=8))     
+```
+
+What are these server ports listening for?
+
+- `8440/tcp`: the Edge Controller listening for API requests. We specified this port as env var `ZITI_EDGE_CONTROLLER_PORT`. You need a firewall exception for this port to allow Ziti identities to obtain an Edge session.
+- `8441/tcp`: the Edge Controller listening for Edge Routers. We specified this port as env var `ZITI_CTRL_PORT`. You need a firewall exception because all routers, including the one running on the same host, will connect to the global advertised address of the controller.
+- `8442/tcp`: the Edge Router listening for edge connections from identities. We specified this port as env var `ZITI_EDGE_ROUTER_PORT`. You need a firewall exception for this port to allow Ziti identities to connect to Ziti services via the Edge Router.
+- `10080/tcp`: the Ziti fabric link listener described below in [Additional Routers](#additional-routers)
+
 ## Adding Environment Variables Back to the Shell
 
 If you log out and log back in again you can source the '.env' file located at:
-`. ~/.ziti/quickstart/$(hostname)/$(hostname).env`. Below is an example of logging in, not having ZITI_HOME set, sourcing the
+`source ~/.ziti/quickstart/$(hostname)/$(hostname).env`. Below is an example of logging in, not having ZITI_HOME set, sourcing the
 .env file and then showing ZITI_HOME set:
 
 ```bash
-Last login: Thu May 20 11:36:25 2021 from 67.246.244.61
-ubuntu@ip-10-0-0-1:~$ echo $ZITI_HOME
-
-ubuntu@ip-10-0-0-1:~$ . ~/.ziti/quickstart/$(hostname)/$(hostname).env
-
+$ source ~/.ziti/quickstart/$(hostname)/$(hostname).env
 adding /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/ziti-bin/ziti-v0.20.2 to the path
-ubuntu@ip-10-0-0-1:~$ echo $ZITI_HOME
+
+$ echo $ZITI_HOME
 /home/ubuntu/.ziti/quickstart/ip-10-0-0-1
 ```
 
-## Install Ziti Admin Console (ZAC) [Optional]
+## Install Ziti Admin Console (ZAC)

```

Comment:
I don't think these new words add any additional clarity to the words that were there before but I could see a "Next Steps" type of section though. more on that below...